components/buttons/buttons.config.yml

```diff
@@ -0,0 +1,8 @@
+status: 'wip'
+context:
```

Comment:
It does not seem like any of these variables are used? if they aren't nessecary please remove them as they add unneeded complexity

Reply:
resolved